Source/Core/BoundingSphere.js

```diff
@@ -292,11 +292,17 @@ define([
         ellipsoid = defaultValue(ellipsoid, Ellipsoid.WGS84);
         surfaceHeight = defaultValue(surfaceHeight, 0.0);
 
-        var positions;
-        if (defined(rectangle)) {
-            positions = Rectangle.subsample(rectangle, ellipsoid, surfaceHeight, fromRectangle3DScratch);
+        if (!defined(result)) {
+            result = new BoundingSphere();
+        }
+
+        if (!defined(rectangle)) {
```

Comment:
`rectangle` is required, right?  Shouldn't this throw a `DeveloperError`?

Reply:
It says `rectangle` is required in the documentation, but it wasn't required in the code.  Should I add a `DeveloperError`?  Would that be a breaking change?

Reply:
Hmm, throughout the `from` functions in this file, it looks like the input is optional...this isn't consistent with `from` functions elsewhere.  So you could either "fix" with a doc update by adding `[]` or more deeply investigate to see if this is really the desired behavior (e.g., due to common undefined input from files) or if it was an oversight, and then make the breaking change.

Reply:
I decided to just update the doc since pretty much all the `from` functions in this file didn't require the input